W-14027157 exchange asset types gr (#371)
mulesoft/docs-gateway: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc

* <<release, Release a Custom Policy>> +
Release the policy to create a definitive version of the policy. After releasing the policy, you must create a new policy version to edit the policy.

> Everytime you publish or release a policy, you create a new `stable` asset in Exchange. For more information about `stable` state assets, see xref:exchange::lifecycle.adoc#stablet-state[Stable State Assets].

Reviewer: ```suggestion
Every time you publish or release a policy, you create a new `stable` asset in Exchange. For more information about `stable` state assets, see xref:exchange::lifecycle.adoc#stablet-state[Stable State Assets].
```

[[publish]]
== Publish a Custom Policy

To publish a `development` state policy asset, run the `publish` command from the policy's root folder:

[source,ssh]
----
make publish
----

After publishing the asset, a new `development` state asset appears in Exchange:

> image:pdk-dev-asset.png["Development assets appear in Exchange labled DEV."]

Reviewer 2: ```suggestion
image:pdk-dev-asset.png["Development assets appear in Exchange labeled DEV."]
```

make publish
----

> After publishing the asset, a new asset labled `DEV` appears in Exchange:

Reviewer 2: ```suggestion
After publishing the asset, a new asset labeled `DEV` appears in Exchange:
```

> image:pdk-dev-asset.png["Policy appear in Exchange labeled DEV."]

Reviewer: ```suggestion
image:pdk-dev-asset.png["Policy appears in Exchange labeled DEV."]
```

make release
----

> After releasing the asset in Exchange, the `DEV` label is removed. The asset's GAV appears similar to the following:

Reviewer 2: The label is not removed from previously published assets. This is a different asset that is being uploaded.
